package.json

```
@@ -2,20 +2,19 @@
   "name": "iam-client-lib",
   "version": "8.0.0-alpha.4",
   "description": "Library for Decentralized Identity and Access Management",
-  "main": "dist/index.js",
-  "module": "dist/index.esm.js",
-  "types": "dist/src/index.d.ts",
+  "main": "dist/lib/index.js",
+  "module": "dist/lib.esm/index.js",
+  "types": "dist/lib/src/index.d.ts",
```

I am more convinced that `lib` is the right directory to put files that would be consumed by a package manager like `npm` 👍 . `dist` seems to be the directory for a bundled file for webpage or CDNs to load directly.
However, given this, I think that the `lib` directory should not be under the `dist` directory. I think `lib` should be at the top level.
```suggestion
  "main": "lib/index.js",
  "module": "lib.esm/index.js",
  "types": "lib/src/index.d.ts",
```